src/argilla/client/feedback/dataset/base.py

```diff
@@ -161,7 +164,7 @@ def __init__(
         # self._extra_metadata_allowed = extra_metadata_allowed
 
     @property
-    @abstractproperty
+    @abstractmethod
     def records(self) -> Any:
         """Returns the records of the dataset."""
         pass
```

Comment:
Just as a side-comment, we should probably briefly discuss which standard do we want to apply for the `@property` + `@abstractmethod` and for the `@abstractmethod` as in some cases we're just adding the docstring, `pass`, `...` or `raise NotImplementedError`, so I think it would be nice to align here.

My personal preference is `...` or `pass`